server/webapp/WEB-INF/rails.new/spec/controllers/admin/plugins/plugins_controller_spec.rb

```diff
@@ -53,7 +63,7 @@
 
     it "should show error message when upload is unsuccessful" do
       @plugin_manager.should_receive(:addPlugin).with(an_instance_of(java.io.File), 'plugins_controller_spec.rb')
-        .and_return(@plugin_response = double('upload_response'))
+      .and_return(@plugin_response = double('upload_response'))
```

Comment:
Place the . on the previous line, together with the method call receiver.<br>Use 2 (not 0) spaces for indenting an expression spanning multiple lines.